Snakefile

```diff
@@ -0,0 +1,63 @@
+
+
+rule all:
+    input:
+        required_outputs=
+
+
+rule remove_old_annotations:
+    """
+    Remove old annotations from Solr for a species
+    """
+    log:
+    input:
+    output:
+    shell:
+    """
+    """
+
+rule prep_working_dirs:
+    """
+    Prepare app and working directories
+    """
+    conda: "envs/cli.yaml"
+    log:
+    input:
+    output:
+    shell:
+    """
+    ./gradlew :cli:bootJar
+    mkdir ./bioentities ./bulk-analytics
+    """
```

Comment:
This is not needed, since the conda package for web-bulk-cli already provides the JAR built.
```suggestion
```